lighty-examples/lighty-bgp-community-restconf-app/src/main/java/io/lighty/examples/controllers/bgpapp/Main.java

```diff
@@ -157,10 +159,10 @@ public synchronized void stop() {
         running = false;
     }
 
-    private static boolean startLightyModule(final LightyModule lightyModule) throws InterruptedException,
+    private boolean startLightyModule(final LightyModule lightyModule) throws InterruptedException,
```

Comment:
This method can still be `static` - just add parameter :)

Reply:
done